homeassistant/components/androidtv/media_player.py

```diff
@@ -157,9 +157,9 @@ def service_adb_command(service):
             output = target_device.adb_command(cmd)
 
             # log the output if there is any
-            if output and (not isinstance(output, str) or output.strip()):
```

Comment:
Is this change just code cleanup, or can it change the behavior?

Reply:
It won't change the behavior. In the `adb_command` I made sure that the return value is a string, so I simplified the logic here.